src/operator/ResourcesAccountType.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  ComponentColor,
+  DropdownMenuTheme,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {OperatorContext} from './context/operator'
+import {AccountType} from 'src/types'
+
+const ResourcesAccountType: FC = () => {
+  const {accountTypes, setAccountTypes} = useContext(OperatorContext)
+  const availableTypes: AccountType[] = allAccountTypes
+
+  const handleSelect = (selectedOption: string): void =>
+    accountTypes.includes(selectedOption as AccountType)
```

Comment:
```suggestion
  const handleSelect = (selectedOption: AccountType): void =>
    accountTypes.includes(selectedOption)
```

Reply:
Because the `onSelect` event returns a string, @asalem1 offered [this solution](https://github.com/influxdata/ui/pull/4218#discussion_r846298246) to this.